recipes/clasv/meta.yaml

```diff
@@ -0,0 +1,154 @@
+{% set name = "CLASV" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/JoiRichi/CLASV/archive/refs/tags/v1.0.3.tar.gz
+  sha256: 2d63625311625fd814eb9293721784deeb4550de0464e5030ae927e4f3f87731
+
+build:
+  number: 0
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin="x.x") }}
+  script:
+    - echo "Contents of work directory:"
+    - set +e
+    - chmod +x $HOME/.local/bin/nextclade
```

Comment:
why is this needed? This should not be install the package into the HOME but in the site-package dir for conda. Please have a look at other Python packages.